woostarter/patterns/about-gallery-row-paragraph-button.php

```diff
@@ -0,0 +1,61 @@
+<?php
+/**
+ * Title: Gallery row with paragraph and button
+ * Slug: woostarter/gallery-row-paragraph-button
+ * Categories: text, about, media
+ * Keywords: about, text, intro
+ * Description: A section with a gallery and an introduction about a store.
+ *
+
+?>
+<!-- wp:group {"metadata":{"categories":["intro"],"name":"Gallery row with paragraph and button","patternName":"a8c/intro-two-column-with-image-and-content"},"align":"full","className":"alignfull is-style-default","style":{"spacing":{"margin":{"top":"0","bottom":"0"},"padding":{"top":"0","bottom":"0"}}},"layout":{"type":"constrained","justifyContent":"center"}} -->
+<div class="wp-block-group alignfull is-style-default" style="margin-top:0;margin-bottom:0;padding-top:0;padding-bottom:0"><!-- wp:spacer {"height":"var:preset|spacing|40"} -->
+<div style="height:var(--wp--preset--spacing--40)" aria-hidden="true" class="wp-block-spacer"></div>
+<!-- /wp:spacer -->
+
+<!-- wp:columns {"isStackedOnMobile":false,"align":"wide","style":{"spacing":{"blockGap":{"left":"var:preset|spacing|20"}}}} -->
+<div class="wp-block-columns alignwide is-not-stacked-on-mobile"><!-- wp:column {"verticalAlignment":"stretch","width":"25%"} -->
+<div class="wp-block-column is-vertically-aligned-stretch" style="flex-basis:25%"><!-- wp:image {"id":331,"aspectRatio":"1","scale":"cover","style":{"color":[]}} -->
+<figure class="wp-block-image"><img src="https://kiri.mystagingwebsite.com/wp-content/uploads/2024/12/about-business-image-859x1024.webp" alt="Antique bottles in a dimly lit arched alcove with a textured wall." class="wp-image-331" style="aspect-ratio:1;object-fit:cover"/></figure>
```

Comment:
This `src` attribute needs to be updated to point to the themes assets directory (`<?php echo esc_url( get_template_directory_uri() ); ?>/assets/images/about-business-image-859x1024.webp`) before merging.